Week-6/Ex3.js

```diff
@@ -0,0 +1,29 @@
+//SC: O(1)
+//TC: O(n)
+function sort012(arr) {
+    let i = 0;
+    let iteration = 0;
+    while(iteration < arr.length) {
```

Comment:
This line will throw error if I pass the array empty.

Build a habit of handling edge cases